### PR TITLE
update maintain-tidb-using-tiup.md

### PR DESCRIPTION
**Pull request by together-wang**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
add Update component

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: maintain-tidb-using-tiup.md [hunk 1]
```diff
@@ -124,6 +124,51 @@ server_configs:
 
 然后执行 `tiup cluster reload ${cluster-name} -N tidb` 命令滚动重启。
 
+## 更新组件
+
+常规的升级集群可以使用 upgrade 命令，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 patch 命令：
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:129]: 是否有必要给出具体的 `upgrade` 命令？

File: maintain-tidb-using-tiup.md [hunk 2]
```diff
@@ -124,6 +124,51 @@ server_configs:
 
 然后执行 `tiup cluster reload ${cluster-name} -N tidb` 命令滚动重启。
 
+## 更新组件
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:127]:
```suggestion
## 升级 TiUP 组件
```

File: maintain-tidb-using-tiup.md [hunk 3]
```diff
@@ -124,6 +124,51 @@ server_configs:
...
+      --overwrite              在未来的 scale-out 操作中使用当前指定的临时包
+  -R, --role strings           指定被替换的服务类型
+      --transfer-timeout int   transfer leader 的超时时间
+
+Global Flags:
+      --ssh-timeout int   SSH 连接的超时时间
+  -y, --yes               跳过所有的确认步骤
+```
+
+例如，有一个 TiDB 的 hotfix 包放在 `/tmp/tidb-hotfix.tar.gz`，如果此时想要替换集群上的所有 TiDB，则可以执行：
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:155]:
```suggestion
例如，有一个 TiDB 的 hotfix 包放在 `/tmp/tidb-hotfix.tar.gz` 目录下。如果此时想要替换集群上的所有 TiDB，则可以执行以下命令：
```

Comment (lilin90) [reply maintain-tidb-using-tiup.md:155]: 这里的`替换`是指升级吗？

File: maintain-tidb-using-tiup.md (hunk 2 above)
Comment (kissmydb) [reply maintain-tidb-using-tiup.md:127]: 这节是说 ： Hotfix 版本替换，吧。标题可以改的明确一点

File: maintain-tidb-using-tiup.md (hunk 1 above)
Comment (kissmydb) [reply maintain-tidb-using-tiup.md:129]: 改成：常规的升级集群请参考 '升级文档的链接' 吧

File: maintain-tidb-using-tiup.md [hunk 4]
```diff
@@ -124,6 +124,50 @@ server_configs:
 
 然后执行 `tiup cluster reload ${cluster-name} -N tidb` 命令滚动重启。
 
+## Hotfix 版本替换
+
+常规的升级集群可以使用 upgrade 命令，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 patch 命令：
```
Comment (together-wang) [inline maintain-tidb-using-tiup.md:129]:
```suggestion
常规的升级集群请参考[升级文档](https://github.com/pingcap/docs-cn/blob/master/tiup/tiup-cluster.md#%E5%8D%87%E7%BA%A7%E6%93%8D%E4%BD%9C) ，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 patch 命令：
```

Comment (together-wang) [inline maintain-tidb-using-tiup.md:129]:
```suggestion
常规的升级集群请参考 [升级文档](https://github.com/pingcap/docs-cn/blob/master/tiup/tiup-cluster.md#%E5%8D%87%E7%BA%A7%E6%93%8D%E4%BD%9C)，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 patch 命令：
```

File: maintain-tidb-using-tiup.md [hunk 5]
```diff
@@ -124,6 +124,50 @@ server_configs:
 
 然后执行 `tiup cluster reload ${cluster-name} -N tidb` 命令滚动重启。
 
+## Hotfix 版本替换
+
+常规的升级集群请参考 [升级文档](https://github.com/pingcap/docs-cn/blob/master/tiup/tiup-cluster.md#%E5%8D%87%E7%BA%A7%E6%93%8D%E4%BD%9C)，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 patch 命令：
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:129]:
```suggestion
常规的升级集群请参考 [升级文档](/tiup/tiup-cluster.md#升级操作)，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 `patch` 命令：
```

File: maintain-tidb-using-tiup.md [hunk 6]
```diff
@@ -124,6 +124,50 @@ server_configs:
...
+      --overwrite              在未来的 scale-out 操作中使用当前指定的临时包
+  -R, --role strings           指定被替换的服务类型
+      --transfer-timeout int   transfer leader 的超时时间
+
+Global Flags:
+      --ssh-timeout int   SSH 连接的超时时间
+  -y, --yes               跳过所有的确认步骤
+```
+
+例如，有一个 TiDB 的 hotfix 包放在 `/tmp/tidb-hotfix.tar.gz` 目录下。如果此时想要替换集群上的所有 TiDB，则可以执行以下命令：
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:155]:
```suggestion
例如，有一个 TiDB 实例的 hotfix 包放在 `/tmp/tidb-hotfix.tar.gz` 目录下。如果此时想要替换集群上的所有 TiDB 实例，则可以执行以下命令：
```

File: maintain-tidb-using-tiup.md [hunk 7]
```diff
@@ -124,6 +124,50 @@ server_configs:
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -R tidb
+```
+
+或者只替换其中一个 TiDB：
```
Comment (lilin90) [inline maintain-tidb-using-tiup.md:163]:
```suggestion
或者只替换其中一个 TiDB 实例：
```

**Comment by lilin90** [comment]
@together-wang @King-Dylan 请确认下我加的最后两条 comment 是否恰当~

File: maintain-tidb-using-tiup.md [hunk 8]
```diff
@@ -124,6 +124,50 @@ server_configs:
 
 然后执行 `tiup cluster reload ${cluster-name} -N tidb` 命令滚动重启。
 
+## Hotfix 版本替换
+
+常规的升级集群请参考 [升级文档](/tiup/tiup-cluster.md#升级操作)，但是在某些场景下（例如 Debug)，可能需要用一个临时的包替换正在运行的组件，此时可以用 `patch` 命令：
```
Comment (yikeke) [inline maintain-tidb-using-tiup.md:129]:
```suggestion
常规的升级集群请参考[升级文档](/tiup/tiup-cluster.md#升级操作)，但是在某些场景下（例如 Debug），可能需要用一个临时的包替换正在运行的组件，此时可以用 `patch` 命令：
```

**Pull request merged**